docs/04_glossary.md

```diff
@@ -33,15 +40,16 @@ The Go client doesn't use this.
 
 ### Decision
 Any action taken by the workflow durable function is called a decision. For example:
-scheduling an activity, canceling a child workflow, or starting a timer.
+scheduling an activity, canceling a child workflow, or starting a timer. A [decision task](completion) contains an optional list of decisions. Every decision is recorded in the [event history](#event-history) as an [event](#event)
 
 ### Decision Task
-
-Every time a new external event that might affect a workflow state is recorded. a decision task
-notifies a workflow worker about it. After the new event is handled, the decision task is completed.
+Every time a new external event that might affect a workflow state is recorded a decision task that contains it is added to a [decision-task-list] and then picked up by a workflow worker. After the new event is handled, the decision task is completed with a list of [decisions](#decision)
```

Comment:
`[decision-task-last]` -> `[decision task list](#decision-task-list)`

Reply:
A comma after "is recorded" would be nice.
same comment about ending with a period.